src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import { Tooltip } from './Tooltip';
+
+describe('<Tooltip />', () => {
```

Comment:
Please add an accessibility test here. You can find an example of how to do this in the [updated testing template](https://github.com/Redback-Operations/redback-ui/blob/main/templates/TemplateName.test.tsx).

Reply:
Added the accessibility test.